src/test/java/com/moodysalem/TimeZoneMapperTest.java

```diff
@@ -75,25 +73,19 @@ public String getExpectedTimezone() {
     public void runTestCases() {
         final AtomicInteger succeeded = new AtomicInteger(0), failed = new AtomicInteger(0);
 
-        final Consumer<TestData> tester = (d) -> {
-            String timezone = TimezoneMapper.tzNameAt(d.getLat(), d.getLng());
-            ZoneId zone = TimezoneMapper.tzAt(d.getLat(), d.getLng());
+        for (TestData data: testData) {
+            String timezone = TimezoneMapper.tzNameAt(data.getLat(), data.getLng());
 
-            String error = String.format(EXPECT_ERROR, d.getName(), d.getLat(), d.getLng(), d.getExpectedTimezone(), timezone);
+            String error = String.format(EXPECT_ERROR, data.getName(), data.getLat(), data.getLng(),
+                    data.getExpectedTimezone(), timezone);
             try {
-                assertTrue(error, d.getExpectedTimezone().equals(timezone));
-                assertTrue(error, ZoneId.of(d.getExpectedTimezone()).equals(zone));
+                assertTrue(error, data.getExpectedTimezone().equals(timezone));
                 succeeded.incrementAndGet();
             } catch (AssertionError e) {
                 failed.incrementAndGet();
                 LOG.warning(e.getMessage());
             }
-        };
-
-        testData.forEach(tester);
-
-        // hand entered test cases
-        tester.accept(new TestData("Chicago, IL", 41.8369, -87.6847, "America/Chicago"));
```

Comment:
i see now why this was added to the test cases-the reason it's separate from the test-cases file is because that test-cases file is from an external source.

btw, a nice improvement we could do here would be to 'unroll' the tests so each latlng test case is actually considered a test case by testng, rather than programmatically aggregating the test results- something i've seen in languages like scala. not sure how easy that is.